build.proj

```diff
@@ -21,8 +21,7 @@
 
   <ItemGroup>
     <Project Include="external\dir.proj" />
-    <Project Include="src\ref.builds" />
```

Comment:
Why does combining these help with this problem? I would still like to keep ref/src logically independent.